airbyte-webapp/src/components/Modal/Modal.tsx

```diff
@@ -22,10 +22,10 @@ const cardStyleBySize = {
 };
 
 const Modal: React.FC<ModalProps> = ({ children, title, onClose, clear, closeOnBackground, size }) => {
-  const handleUserKeyPress = useCallback((event, closeModal) => {
-    const { keyCode } = event;
+  const handleUserKeyPress = useCallback((event: KeyboardEvent, closeModal: () => void) => {
+    const { key } = event;
```

Comment:
:information_source: Changed this to use `key` instead of the deprecated `keyCode`, so I could test it via `@testing-library/user-events`.